spec/bindgen/parser/type_spec.cr

```diff
@@ -53,7 +53,7 @@ describe Bindgen::Parser::Type do
       type.reference?.should be_false
       type.pointer.should eq(1)
       type.base_name.should eq("int")
-      type.full_name.should eq("int*")
+      type.full_name.should eq("int *")
```

Comment:
Where's the extra space coming from?

Reply:
Pointers/references coming from the Clang parser have the extra space, so `Parser::Type.parse` now canonicalizes the typename given to it with `Cpp::Typename`, instead of using the typename verbatim. (This probably fixed a bug somewhere that considered `T*` and `T *` to be unequal types.)